### PR TITLE
[Added Websites] Ads i came accross while watching Anime

### PR DESCRIPTION
**Pull request by tobi-blazor**
## Summary
Came accross some Ads while watching anime. Wanted to make them available for everybody.

## Checklist

- [X] I have verified that I have not modified the following files:
      - inside the `adguard` folder
      - inside the `alt-version` folder
      - inside the `dnsmasq-version` folder
      - everything.txt

// First PR ever, yeyyy

### CODE REVIEW

File: ads.txt [hunk 1]
```diff
@@ -154719,3 +154719,14 @@
 0.0.0.0 torimochi.line-apps.com
 0.0.0.0 aax-us.amazom-adsystem.com
 0.0.0.0 aax-us-pdx.amazom-adsystem.com
+0.0.0.0 topmusicalcomedy.com
+0.0.0.0 crystal-blocker.com
+0.0.0.0 platincasino.com
```
Comment (spirillen) [inline ads.txt:154724]: Gambling: https://mypdns.org/my-privacy-dns/matrix/-/issues/4122446

File: ads.txt [hunk 2]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 regadspro.com
```
Comment (spirillen) [inline ads.txt:154725]: Redirects....

image

File: ads.txt [hunk 3]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 ezadblocker.com
```
Comment (spirillen) [inline ads.txt:154726]: image

File: ads.txt [hunk 4]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 wyylde.com
```
Comment (spirillen) [inline ads.txt:154727]: NSFW: https://mypdns.org/my-privacy-dns/matrix/-/issues/4122447

File: ads.txt [hunk 5]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 watchadzfree.com
```
Comment (spirillen) [inline ads.txt:154728]: :question: 

image

File: ads.txt [hunk 6]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 watch-adsfree.info
```
Comment (spirillen) [inline ads.txt:154729]: :question: 

image

File: ads.txt [hunk 7]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 vipcaptcha.live
```
Comment (spirillen) [inline ads.txt:154730]: :question: 

image

File: ads.txt [hunk 8]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 updategreatlyoriginalinfo-file.info
```
Comment (spirillen) [inline ads.txt:154731]: :question: 

image

File: ads.txt [hunk 9]
```diff
@@ -154719,3 +154719,14 @@
...
+0.0.0.0 updategreatlyprogressiveinfo-file.info
```
Comment (spirillen) [inline ads.txt:154732]: :question: 

image

**Comment by tobi-blazor** [comment]
Since many of those Websites are not available anymore, I'm gonna close this PR.

**Pull request closed without merging**